paper.tex

```diff
@@ -63,13 +63,15 @@ \section{Introduction}
 The term “Research software” is also defined somewhat differently within the community.
 Therefore, to avoid ambiguities, we list the ones we use here:\\
 \textbf{Software:}\\
-Source code, documentation, tests, executables and all other artifacts that are created by humans during the development process that are necessary to understand its purpose.\\
+Source code, documentation, tests, executables and all other artefacts that are created during the development process that are necessary to understand its purpose.\\
 \textbf{Research software:}\\
 Foundational algorithms, the software itself, as well as scripts and computational workflows that were created
 during the research process or for a research purpose, across all domains of research.
 This definition is broader than in~\autocite{FAIR4RS} and is the outcome of a recent discussion in~\autocite{Gruenpeter2021}.\\
 \textbf{Research software engineers:}\\
-People who create or improve research software and/or the structures that it interacts with in the computational ecosystem of research domains.
```

Comment:
There is the same "problem" here: somehow your editor adds line breaks where there are supposed to be none.